File: 1 - Sequenciais/sequenciais2.php
<?php #Faça um programa que peça um número e então mostre a mensagem: "O número informado foi {numero}. ?>

/*
* Author:  Igor Vinicius Freitas de Souza
* GitHub: https://github.com/igor1043
* E-mail: igorviniciusfreitasouza@gmail.com
*/

<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Sequenciais 2 - PHP</title>
  </head>
  <body>
    <?php
      if (!empty($_GET))
      {
        echo "O número informado foi: " . $_GET['numero'];
      }
      else
      {
        echo '
        <form action="sequenciais2.php" method="get">
          Digite um número: <input type="text" name="numero"/>
          <input type="submit" value="Enter">
        </form>';
      }
    ?>
  </body>
</html>

File: 1 - Sequenciais/sequenciais2.java
//Faça um programa que peça um número e então mostre a mensagem: "O número informado foi {numero}.

/*
* Author:  Igor Vinicius Freitas de Souza
* GitHub: https://github.com/igor1043
* E-mail: igorviniciusfreitasouza@gmail.com
*/

import java.util.Scanner;

public class sequenciais2 {

  public static void main(String[] args) {
    System.out.println("Digite um número: ");
    Scanner s = new Scanner(System.in);
    int numero = s.nextInt();
    System.out.println("O número digitado foi: " + numero);
  }

}
